### ISSUE TITLE
NTR: Mouse strain B6-Ly5.1

### ISSUE THREAD

**Issue by mobilegenome** [EBISPOT/efo#974]
This is a commonly used laboratory mouse strain, not yet included in the EFO. 

## Preferred term label

B6-Ly5.1

## Synonyms

B6-Cd45.1
B6.SJL-Ptprc<sup>a</sup>Pepc<sup>b</sup>/BoyJ
Pep Boy

## Textual definition

This C57BL/6 congenic strain is used widely in transplant studies because it carries the differential Ptprca pan leukocyte marker commonly known as CD45.1 or Ly5.1. Wild-type C57BL/6 inbred mice express the Ptprcb (CD45.2 or Ly5.2) allele.

The current gene symbol designation Ptprc instead of Cd45 or Ly5 is based on work by Charbonneau and colleagues who showed in humans that a protein-tyrosine phosphatase (human placental protein-tyrosine phosphatase 1B) is homologous to the CD45 protein. Ptprc is one of a family of protein-tyrosine phosphatase genes involved in the regulation of cell growth.

The Ptprca allele and closely linked Pepcb allele were serially backcrossed from SJL/J onto C57BL/6 with selection at each generation for Ptprca by serotyping (Shen et al., 1985). In 1990 this strain, then housed at Sloan Kettering, was imported into The Jackson Laboratory from Edward Boyse at generation N22. Since its arrival, the strain has been maintained by sibling inbreeding. In 2005 it reached generation N22F55.

Source: https://www.jax.org/strain/002014

Reference
PMID: 3864163

## Suggested parent term

http://www.ebi.ac.uk/efo/EFO_0000606

## Attribution

https://orcid.org/0000-0002-3110-8220

**Comment by paolaroncaglia** [comment]
Hi @mobilegenome ,
Thanks for your suggestion. I will add a new EFO term as above in time for the next EFO release. I'll shorten the definition to include only the first paragraph (2 sentences), as the more detailed knowledge can be gathered from the references if needed.

**Comment by paolaroncaglia** [comment]
Hi @mobilegenome ,
I've added your suggested new term as EFO_0011017 B6-Ly5.1. It'll be in the next EFO release.
Thanks,
Paola

**Comment by mobilegenome** [comment]
Thank you!
